pkg/parser/expression.go

```diff
@@ -269,6 +271,34 @@ func (p *parser) parseDotExpr(left Node) Node {
 	return expr
 }
 
+func (p *parser) parseTypeAssertion(left Node) Node {
+	tok := p.cur
+	if p.lookAt(p.pos-1).Type == lexer.WS {
+		p.appendError("unexpected whitespace before '.'")
+		return nil
+	}
+	if p.lookAt(p.pos+1).Type == lexer.WS {
+		p.appendError("unexpected whitespace after '.'")
+		return nil
+	}
+	p.advance() // advance past .
+	p.advance() // advance past (
+	t := p.parseType()
+	switch t {
+	case ILLEGAL_TYPE:
+		p.appendErrorForToken("invalid type in type assertion of '"+left.String()+"'", tok)
+	case ANY_TYPE:
+		p.appendErrorForToken("cannot type assert to type 'any'", tok)
+	}
+	if p.assertToken(lexer.RPAREN) {
+		p.advance() // advance past )
+	}
+	if left.Type() != ANY_TYPE {
+		p.appendErrorForToken("value of type assertion must be of type 'any', not '"+left.Type().String()+"'", tok)
```

Comment:
I think we don't use single quotes anymore and i never quotes types, only var and func names.